ugandaemr_mobile/terminologies.md

```diff
@@ -0,0 +1,12 @@
+# Terminologies
```

Comment:
@solemabrothers This page is not needed as it is available in the UgandaEMR main user manual

Reply:
Done

Reply:
Done